Fix coding style final

thalieht opened the pull request:
It's big but it should be the last one.
The diff for transferlistsortmodel.cpp is messed up on github, it's just indentation changes. 

P.S. After this, if there is something you want done but you consider a chore, and is within my power let me know.

Chocobo1 commented on src/gui/about_imp.h:
```diff
@@ -24,26 +24,25 @@
 #include "utils.h"
 
-class about: public QDialog, private Ui::AboutDlg
+class About : public QDialog, private Ui::AboutDlg
```
`AboutDialog`

Chocobo1 commented on src/gui/properties/peerlistwidget.cpp:
```diff
@@ -271,7 +271,6 @@ void PeerListWidget::showPeerListMenu(const QPoint &)
     }
     if (act == copyPeerAct) {
         copySelectedPeers();
-        return;
```
I think you should preserve it.

Chocobo1 commented on src/gui/properties/trackerlist.cpp:
```diff
@@ -571,7 +571,6 @@ void TrackerList::showTrackerListMenu(QPoint)
     }
     if (act == editAct) {
         editSelectedTracker();
-        return;
```
here too.

Chocobo1 commented on src/gui/transferlistfilterswidget.h:
```diff
@@ -116,10 +111,10 @@ private slots:
-    virtual void applyFilter(int row);
-    virtual void handleNewTorrent(BitTorrent::TorrentHandle *const torrent);
-    virtual void torrentAboutToBeDeleted(BitTorrent::TorrentHandle *const torrent);
+    virtual void showMenu(QPoint) override;
```
can the `virtual` be striped off?

thalieht replied:
Yeah, i'll fix these and the other comments.

Chocobo1 commented:
Please squash commits.

thalieht commented:
Squashed

glassez commented on src/gui/properties/speedplotview.cpp:
```diff
@@ -161,7 +161,7 @@ int SpeedPlotView::maxYValue()
             continue;
 
-        for (int i = int(queue.size()) - 1, j = 0; i >= 0 && j <= m_viewablePointsCount; --i, ++j)
+        for (int i = int(queue.size()) - 1, j = 0; (i >= 0) && (j <= m_viewablePointsCount); --i, ++j)
```
`static_cast<int>`
And below too.

glassez commented on src/gui/torrentcreatordlg.h:
```diff
@@ -44,7 +44,7 @@ namespace BitTorrent
 }
 
-class TorrentCreatorDlg: public QDialog
+class TorrentCreatorDlg : public QDialog
```
TorrentCreatorDialog

glassez commented on src/gui/torrentmodel.h:
```diff
@@ -85,12 +83,12 @@ class TorrentModel : public QAbstractListModel
 
     explicit TorrentModel(QObject *parent = nullptr);
```
This legacy name is inappropriate. It should be "TorrentListModel".

glassez replied:
Although, since all related classes use "TransferList" term, it should be TransferListModel.

glassez commented on src/gui/transferlistfilterswidget.h:
```diff
@@ -47,15 +42,15 @@ namespace BitTorrent
 }
 
-class FiltersBase: public QListWidget
+class FiltersBase : public QListWidget
```
BaseFilterWidget

glassez commented on src/gui/transferlistfilterswidget.h:
```diff
@@ -70,7 +65,7 @@ private slots:
 };
 
-class StatusFiltersWidget: public FiltersBase
+class StatusFiltersWidget : public FiltersBase
```
It seems like plural form is inappropriate in these class names. It should be StatusFilterWidget and so on.

glassez commented on src/gui/transferlistfilterswidget.h:
```diff
@@ -138,7 +133,7 @@ private slots:
 class TagFilterWidget;
 
-class TransferListFiltersWidget: public QFrame
+class TransferListFiltersWidget : public QFrame
```
Here plural is correct since this widget is container for several filter widgets.

thalieht commented:
Rebased and addressed some of the comments. I'll do the dlg->dialog renames in another PR and the names in .ui files in another because it probably won't be small.

About the names in .ui, is it worth it if Designer uses snake_case and most of the changes/new files are done in it? Seems like an uphill battle. 
If you still want it, do you have any preferences about the naming? i mean put a prefix or suffix to the different types e.g checkbox names chkBoxName1 chkBoxName2 or something along these lines. 
In another PR i made a few changes in some file (don't remember which) to the names and i changed the label names to labelName1 labelName2 etc.

glassez commented:
>Rebased and addressed some of the comments.

Ok. Squash your commits and I'll approve it.

>About the names in .ui, is it worth it if Designer uses snake_case and most of the changes/new files are done in it? Seems like an uphill battle.

I'm not asking you to change all names. Only the ones we use in the code.
>do you have any preferences about the naming? i mean put a prefix or suffix to the different types e.g checkbox names chkBoxName1 chkBoxName2 

I prefer prefixes for variable names (e.g. labelName1, checkBoxName2, buttonName3 etc.).

thalieht commented:
Squashed.
>Only the ones we use in the code.

Great this shouldn't be much work then i'll get right to it.

glassez commented:
Thank you!

thalieht commented:
Thanks
